### DISCUSSION TITLE
How to create an project in vercel?

### DISCUSSION THREAD

**Discussion by adityamittal3107** [vercel/community#4271] [Help]
### Question

Hi team, I am creating an integration of my company in vercel. In the setup wizard, I am using create Project vercel API to create a new project but it is throwing 403 error and says you do not have permission to create a project. What permission do we need for creating a project? Here is my code snippet - 
`
const createProject = await fetch(
        `https://api.vercel.com/v9/projects?teamId=${teamId}`,
        {
          method: 'POST',
          headers: {
            Authorization: `Bearer ${accessToken}`,
          },
          body: JSON.stringify({
            name: 'token-auth-service',
            buildCommand: 'npm run build',
            installCommand: 'npm install',
            outputDirectory: 'public',
            environmentVariables: [
              {
                key: 'TS_SECRET_KEY',
                value: secretKey,
                type: 'encrypted',
                target: ['production', 'preview'],
              },
            ],
            gitRepository: {
              repo: 'https://github.com/girish-thoughtspot/token-auth-service',
              type: 'github',
            },
          }),
        }
      );
`

**Comment by amyegan** [comment]
Hi @adityamittal3107! You need to [create an Access Token](https://vercel.com/docs/rest-api#creating-an-access-token) with the right scope to access your team and project. 

When [accessing resources owned by a Team](https://vercel.com/docs/rest-api#accessing-resources-owned-by-a-team), you need to make sure you pass the correct `teamId` value as a query parameter. If the value does not match a team that you belong to, then you will see a "not authorized" error message.

**Comment by adityamittal3107** [comment]
Hi @amyegan I am passing teamID and vercel authToken. All of my vercel APIs are working except this. Is there any integration setting that we have to set to make it work?

**Reply by adityamittal3107** [reply]
Hi @amyegan Any update?

**Reply by adityamittal3107** [reply]
Hi @amyegan This is very important for us. Can someone please help on this on priority?
